Makefile

```diff
@@ -52,9 +52,10 @@ calico_test/.calico_test.created: $(TEST_CONTAINER_FILES)
 	cd calico_test && docker build -t calico/test:latest .
 	touch calico_test/.calico_test.created
 
-calico_node/.calico_node.created: $(NODE_CONTAINER_FILES)
-	cd calico_node && docker build -t calico/node:latest .
-	touch calico_node/.calico_node.created
+calico_node/docker_dist/.calico_node.created: $(NODE_CONTAINER_FILES)
+	cd calico_node && docker build -f docker_dist/Dockerfile -t calico/node:latest .
```

Comment:
I thought the final argument for `docker build` (`.` ) would be the context root